.circleci/config.yml

```diff
@@ -7,6 +7,32 @@ defaults: &defaults
 
 version: 2
 jobs:
+  linkchecker:
+    <<: *defaults
+    steps:
+      - checkout
+      - run:
+          name: Install Linkchecker
+          command: apt-get install -y linkchecker
+      - run:
+          name: Yarn install
+          command: yarn install
+      - run:
+          name: Yarn build
+          command: yarn build
+      - run:
+          name: Install dependencies
+          command: pip3 install -r requirements.txt
+      - run:
+          name: Start development server
+          command: ./entrypoint 0.0.0.0:8029
+          background: true
+      - run:
+          name: Wait for development server
+          command: sleep 5
```

Comment:
This isn't very resilient, it should be polling instead.

Reply:
This is how it was set up on www.ubuntu.com - if you think you can improve it, please submit a PR when it lands.